### PR TITLE
History record

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

File: bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepo.kt [hunk 1]
```diff
@@ -545,4 +556,18 @@ data class ScheduledNotification(
...
+data class AssessmentHistoryRecord (
+    val instanceGuid: String,
+    val assessmentInfo: AssessmentInfo,
+    val startedOn: Instant,
+    val finishedOn: Instant,
+    val clientTimeZone: String? = null,
+    val clientData: kotlinx.serialization.json.JsonElement? = null) {
+
+    //TODO: Switch to using Instant math once we migrate to Kotlin 1.5 -nbrown 07/27/21
+    val minutes: Int = ((finishedOn.epochSeconds - startedOn.epochSeconds) / 60).toInt()
```
Comment (syoung-smallwisdom) [inline bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepo.kt:570]: `toInt()` will floor the value, when really we want to round it.

**Review by syoung-smallwisdom — COMMENTED** [review]
Looks good. One comment about

**Review by syoung-smallwisdom — APPROVED** [review]
One comment about rounding. Otherwise, looks good

File: bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepo.kt (hunk 1 above)
Comment (nategbrown9) [reply bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepo.kt:570]: The division operation is already doing a floor, `toInt()` is just converting the long into an int. In kotlin `long / long = long`. As soon as we move to kotlin 1.5.x (which I would like to do before our October release), we can use `val minutes = finishedOn.minus(startedOn).inMinutes.roundToInt()`

Comment (syoung-smallwisdom) [reply bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepo.kt:570]: Ah, didn't realize epochSeconds was a Long. (In iOS it's a Double)

**Pull request merged**